### ISSUE TITLE
Cant verify my device

### ISSUE THREAD

**Issue by TooManyDots** [vector-im/element-web#15539]
### Description

I cant verify devices

### Steps to reproduce

- Add a second device
- Get message that I need to verify my device
- Dont know what to do since your ui is unclear

Describe how what happens differs from what you expected.

I dont know what to do in this screen:
![image](https://user-images.githubusercontent.com/35289254/97029704-18ac8280-155e-11eb-807c-02c2c7f12fd4.png)
Nothing is clickable or interactable except the X and skip button. This user experience is horrible.

### Version information
- **Platform**: web in-browser

For the web app:
- **Browser**: Firefox 82.0 ( 64 bit )
- **OS**: Windows
- **URL**: private server, i dont know which version

**Comment by github-tomster** [comment]
@toomanydots
indeed i agree, it is confusing!
let's say: it is a feature, not a bug though.
an attempt of an explanation: 
if you use matrix from different sessions/sources/devices (pc,smartphone,web)
you have to verify each new one to use it seamless across all your sessions.
the action takes place on your other active session, if you have one...
only then, if you have an other active session, you see the message:
'was that you? verify the other session!'
there you have to initiate the verification steps.

**Comment by TooManyDots** [comment]
I got no message on my other computer.

**Comment by github-tomster** [comment]
@toomanydots
try to change your point of view to: matrix is all about security!
it will make thing easier at all.
unfortunately the code is still early beta and not stable at all.
take a look at all the issues, especially the security/key related ones, at element-web or synapse.
try to accept the whole matrix as an WiP.
the shiny matrix-website appearances is deceiving.
it says: 
As of June 2019, Matrix is out of beta, and the protocol is fully suitable for production usage.
lol, no comment!
just look at the issues on github. just another wip-project.

however, you can try to only create non-encrypted rooms/chats so no messages
get encrypted at all and you will be able to follow your chats from any device/session.
and you can ignore the verify-warning.

tl;dr maybe matrix is not the communication platform that fits you...

**Comment by jryans** [comment]
@github-tomster Please do not assume such bad faith... 😭 There are many thousands of suggestions, requests, and bugs that all need to be prioritised.

**Comment by jryans** [comment]
@TooManyDots Did you close this because it is resolved, or out of frustration...? Assuming it's still an issue for you, let's work out how we can resolve it.

**Comment by jryans** [comment]
For example, if you never received a notification on your other session, let's work out how to fix that. If this dialog is to confusion, let's work out how it might be improved.

**Comment by TooManyDots** [comment]
Frustration. I still can't or don't know how to verify my device. For now I solved this issue by using ublock origin to block this notifcation.

I received no notifcation on any other device, neither my other pc or my phone.

**Comment by jryans** [comment]
@TooManyDots If you are able to submit debug logs from each device, we can try to investigate. I know it's frustrating when things like this happen, but let's try to improve it for the next person.

**Comment by TooManyDots** [comment]
I dont know how to find debug logs.

**Comment by jryans** [comment]
@TooManyDots I think it would be most interesting to know why your already logged in sessions did not show a notification, so for a web session, go to Top Left Menu -> Settings -> Help -> Submit debug logs. On your phone, there should be a similar process in the Settings menu.

**Comment by jryans** [comment]
Actually, probably best to have logs from both an existing session and the new one for comparison. So, on the screen above, you could skip for now and get into the app where debug logs can be submitted.

**Comment by TooManyDots** [comment]
In the logs are both tar files. pc1 is my old pc on which i've been using element for at least half a year now. pc2 is my new pc on which i logged into element for the first time like two days ago.

What makes no sense to me is that I am getting the "verify this device"-message on pc1, meaing my oldest login.

I can't find a way to download the logs on my phone. I enabled dev-option, shook my phone, it asked me wether I wanted to make a new (github?)issue and I got the option to upload the logs, but no option to just get the logs.

[Logs](https://github.com/vector-im/element-web/files/5439989/we_dont_support_that_file_type_try_again_with_gif_jpeg_png_docx_gzu_log_pdf_pptx_txt_slsx_or_zip.zip)

**Comment by jryans** [comment]
Thanks, I'll take a look in more detail soon, most likely tomorrow.

**Comment by jywarren** [comment]
Hi, just noting that I had some uncertainty on this menu as well on my laptop, and my phone did not show a notification to verify. I wasn't sure if i was supposed to see it there, though, and thought about possible UI refinements:

1. perhaps a spinning icon and a notice like `check your other devices for a notification prompting you to verify this session`
2. clarify if it can be done even without the notification popping up, in a menu? I couldn't find a way.
3. noting that there may not be a way to verify if you have no other device set up to do so (is this true?)

I'm happy to move to #13909 which may be a good place to split out the UI ambiguities vs. a specific functional issue with verification. Thank you!

**Comment by jryans** [comment]
@jywarren Yes, let's please keep this issue focused on why the expected verification did not show up at all, and use https://github.com/vector-im/element-web/issues/13909 for the confusing UX.

**Comment by jryans** [comment]
@TooManyDots Still looking through the logs, no conclusions yet, but a few more comments...

> What makes no sense to me is that I am getting the ~"verify this device"~ "Verify this session"-message on pc1, meaing my oldest login.

You appear to be logging out and back into Element on pc1 very often, as in multiple times a day. Is there a reason for this? Assuming that is actually happening, it explains part of the story: each time you log out and back in, that's treated as new device as far as your Matrix account is concerned.

> I can't find a way to download the logs on my phone. I enabled dev-option, shook my phone, it asked me wether I wanted to make a new (github?)issue and I got the option to upload the logs, but no option to just get the logs.

There may not be such an option yet, the one to downloads on web is quite new. It would be good to look for a feature request for downloading logs on whichever mobile client you use.

**Comment by TooManyDots** [comment]
Is closing a tab logging out? I do that a lot. Otherwise I dont even know where the logout-button is.

**Comment by t3chguy** [comment]
If afterwards you have to enter your username+password again then yes. Do you have your browser set to clear cookies?

**Comment by TooManyDots** [comment]
I dont have to enter my username and password again. I don't have my browser set to clear cookies.

**Comment by jryans** [comment]
@TooManyDots Above you said you are getting the "verify this session" message on pc1. Does that mean the message with the icons for desktop and mobile in your original report above? That message is only shown just after logging in, but you've said you are not logging in over and over... Do you mean you're seeing a different message, perhaps a "toast" in the top left corner? If yes, could you include a screenshot?

**Comment by mrclauss** [comment]
Yeah so is there any way to fix this in the meanwhile? I do encounter the same issue. I don't get anything on any other session.

**Comment by jryans** [comment]
It seems the original reporter has disappeared, so I'm going to close this issue. For those with other issues, please file a new issue and send debug logs.

**Comment by webbb82** [comment]
im also dealing with this issue.  it really keeps me away from using this even though i really really want to use it. just thinking about trying to deal with this having to sign in then sign in then find the notification then go back. and thats if it works.  I dont get the other one half the time, I use 3-4 devices so i dont know what one its going to.  is there really no better option?  or better way I can use it?
